### PR TITLE
feat(theme): buttons

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
Buttons have not a really sufficient accessible contrast ratio

**What is the chosen solution to this problem?**
It's time to fix that ratio and simplify their usage
image

And it was time to align all grays too
image

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_button-groups.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
 		}
 	}
 
-	> .dropdown-menu {
+	.dropdown-menu {
```
Comment (jmfrancois) [inline packages/theme/src/theme/_button-groups.scss:38]: I think this one may break stuff. is it really needed ?
Like dd in dd

**Review by ncomont — APPROVED** [review]

**Review by jmfrancois — COMMENTED** [review]
Some points:
* the button content is not centered (padding on icon)
* i can t test keyboard navigation on buttons (may be because of href="#")
* no more difference on link with focus only btn

File: packages/theme/src/theme/_button-groups.scss [hunk 2]
```diff
@@ -46,7 +46,11 @@
 	&.open {
 		overflow: visible;
 
-		> .dropdown-menu {
+		.dropdown-toggle {
+			box-shadow: none;
+		}
+
+		.dropdown-menu {
```
Comment (frassinier) [inline packages/theme/src/theme/_button-groups.scss:53]:
```suggestion
		> .dropdown-menu {
```

**Review by jsomsanith-tlnd — COMMENTED** [review]
image
Active tab with dropdown make the text has the background color

File: packages/components/src/HeaderBar/HeaderBar.scss [hunk 3]
```diff
@@ -20,6 +20,10 @@ $background-color-on-hover: rgba(255, 255, 255, 0.2);
 	width: 100%;
 	z-index: 1030;
 
+	:global(.btn) {
+		--btn-color: var(--btn-primary-color);
```
Comment (jmfrancois) [inline packages/components/src/HeaderBar/HeaderBar.scss:24]: ❤️

**Review by jmfrancois — APPROVED** [review]
Still some broken example to fix in storybook but it seems good on the overall PR
* http://3252.talend.surge.sh/components/?path=/story/messaging-communication-status--default (hover)
* http://3252.talend.surge.sh/components/?path=/story/layout-collapsiblepanel--default (icons not centered on the right)

**Pull request merged**